pulp_rpm/app/tasks/synchronizing.py

```diff
@@ -263,18 +271,18 @@ def newpkgcb(pkgId, name, arch):
                 done, pending = await asyncio.wait(pending, return_when=asyncio.FIRST_COMPLETED)
                 for downloader in done:
                     results = downloader.result()
-                    if results[0].url == package_repodata_urls['primary']:
+                    if results[0].url == package_repodata_urls["primary"]:
                         primary_xml_path = results[0].path
                         filelists_xml_path = results[1].path
                         other_xml_path = results[2].path
                         metadata_pb.done += 3
                         metadata_pb.save()
 
-                        packages = await RpmFirstStage.parse_repodata(primary_xml_path,
```

Comment:
to me, this is more readable than what black is doing.

Reply:
In my opinion Black is making more readable (both Human and Machine readable)

Black could do better if instead of inline it did one argument by line

```python
packages = await RpmFirstStage.parse_repodata(
    primary_xml_path, 
    filelists_xml_path, 
    other_xml_path
)
```

This style makes easier to run `find-replace` tools and also easier to analyse code in reviews.

Reply:
I agree. It seems like it should either be one line (if it fits) or multiple lines.